experimental/devices/inky/lut.go

```diff
@@ -0,0 +1,222 @@
+package inky
+
+var (
+	blackLUT = []byte{
```

Comment:
Use [...] form so it is an array instead of a slice. See https://blog.golang.org/go-slices-usage-and-internals for more info.